src/index.html

```diff
@@ -2,11 +2,47 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1.0"
+    >
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-day--wed calendar--month-length_29">
```

Comment:
try to use just one rule for naming
```suggestion
    <div class="calendar calendar--start-day_wed calendar--month-length_29">
```
or
```suggestion
    <div class="calendar calendar--start-day--wed calendar--month-length--29">
```